ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs

```diff
@@ -80,7 +80,28 @@ public void Convert()
                 }
             }
         }
+        public static ReflectionItem HandleContentSourceMeta(ReflectionItem item)
```

Comment:
can this be shared with existing method in ECMAStore?

Reply:
Updated